docker/base/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG cert_image
 ARG root_image
 
 FROM $cert_image AS cert
-RUN apk add --update --no-cache ca-certificates
+RUN apk add --update --no-cache ca-certificates mailcap
 
 FROM $root_image
 COPY --from=cert /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
don't we need to add a COPY command here to copy the mime.types files?

Reply:
ohh right. I missed it.